server/database/queries/house/getFavorite.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../../config/connection');
+
+const getFavoriteHouses = (userId) => {
+  const sql = {
+    text:
+      'SELECT h.*, u.* FROM favorites AS f INNER JOIN users AS u ON u.id = f.user_id INNER JOIN houses AS h ON h.id = f.house_id WHERE f.user_id = $1 ',
```

Comment:
We need to be carefull about sending user data such as password 
Send just contact info